samples/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "nodejs-docs-samples-bigquery-storage",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "engines": {
+    "node": ">=8"
+  },
+  "files": [
+    "*.js"
+  ],
+  "scripts": {
+    "test": "c8 mocha --timeout 600000 test/*.js"
+  },
+  "dependencies": {
+    "@google-cloud/bigquery-storage": "^1.0.1",
```

Comment:
I believe this should be `@google-cloud/bigquery-storage": "^0.1.0", to match the current value in the top level `package.json`.